src/media/js/views/search.js

```diff
@@ -208,9 +208,16 @@ define('views/search',
             if (query) break;
         }
 
-        if (query && queryParam === 'author') {
-            pageTypes += ' leaf';
-            utilsLocal.headerTitle(gettext('Developer Listing'));
+        if (query) {
+            switch (queryParam) {
+                case 'author':
+                    pageTypes += ' leaf';
```

Comment:
your pageTypes array has a string with two elements that should be split.

do pageTypes.join(' ') only once at the end